### DISCUSSION TITLE
求助,如何让指定ip绕过openclash核心,

### DISCUSSION THREAD

**Discussion by jt8823** [vernesong/OpenClash#3425] [Q&A]
请教一下
我先一直使用的ssrplus,  刚转到clash,  确实速度这块,要快一些
但是我遇到了一个问题
因为我家智能设备有些多,使用了clash后出现了不稳定情况,
我已经绑定我这些设备的mac ip  指定它们直连,但是还是不行
- SRC-IP-CIDR,192.168.1.1/32,DIRECT
请问有没有办法让指定这些设备不走clash核心

当前使用的fake-ip模式.   谢谢

**Comment by mphin** [comment]
开启绕过中国大陆IP啊，要稳定就redir_host模式

**Comment by duetg** [comment]
在DNS设置中选择使用防火墙转发，这样在黑白名单里就出现了局域网访问有关的控制选项。

**Reply by zenmaster-cd** [reply]
但有个问题不晓得你们遇到没有 就是微软的 egde 用了这个模式后无法上网，不晓得什么问题 用fake ip 模式

**Reply by kid101x** [reply]
Edge内的安全DNS设置问题？
